MANIFEST.in

```diff
@@ -0,0 +1,2 @@
+recursive-include llvm-project **/*
+recursive-exclude llvm-project/.git **/*
```

Comment:
I'm curious: Which tool uses this file?

Reply:
Usually, `setuptools` uses it to include/exclude stuff from the `sdist`, but I don't think `scikit-build-core` uses it at all. Let me test and see if we need it now to produce a working `sdist`.

Reply:
File removed in https://github.com/finch-tensor/Finch-mlir/pull/16/commits/cc282d892244e6135641311d5212712c7174dcf6